src/backend/query_builder.rs

```diff
@@ -1060,6 +1062,34 @@ pub trait QueryBuilder:
             Value::ChronoDateTimeWithTimeZone(Some(v)) => {
                 write!(s, "'{}'", v.format("%Y-%m-%d %H:%M:%S %:z")).unwrap()
             }
+            #[cfg(feature = "postgres-interval")]
+            Value::Interval(Some(v)) => {
+                let mut space = false;
+
+                write!(s, "'").unwrap();
+
+                if v.months > 0 {
+                    write!(s, "{} MONTHS", v.days).unwrap();
+                    space = true;
+                }
+
+                if v.days > 0 {
+                    if space {
+                        write!(s, " ").unwrap();
+                    }
+                    write!(s, "{} DAYS", v.days).unwrap();
+                    space = true;
+                }
+
+                if v.microseconds > 0 {
+                    if space {
+                        write!(s, " ").unwrap();
+                    }
+                    write!(s, "{} MICROSECONDS", v.microseconds).unwrap();
+                }
+
+                write!(s, "'::interval").unwrap();
+            }
```

Comment:
Shall I make the units abbreviated or is this better for clarity when inspecting the built SQL statement?